roles/openshift_common/tasks/atomic_docker_check.yml

```diff
@@ -0,0 +1,16 @@
+---
+# Fail early if Atomic and old version of Docker
+- block:
+
+  - name: Determine available Docker
+    shell: "rpm -q --queryformat '---\ncurr_version: %{VERSION}\n' docker"
```

Comment:
https://github.com/ansible/ansible/issues/15892 looks like it may make using braces a bit rough but I'll try.